vm_setup_vars.yml

```diff
@@ -30,7 +31,12 @@ flavors:
     disk: "{{ lookup('env', 'EXTRA_WORKER_DISK') }}"
     vcpu: "{{ lookup('env', 'EXTRA_WORKER_VCPU') }}"
     extradisks: "{{ lookup('env', 'VM_EXTRADISKS') | default(false) }}"
-
+  
+  fleeting:
+    memory: "{{ lookup('env', 'FLEETING_MEMORY') }}"
+    disk: "{{ lookup('env', 'FLEETING_DISK') }}"
+    vcpu: "{{ lookup('env', 'FLEETING_VCPU') }}"
```

Comment:
Assuming we can't just use one of the masters, you could default these to e.g `MASTER_MEMORY` etc